media-libs/opencolorio/opencolorio-2.1.2.ebuild

```diff
@@ -55,6 +55,7 @@ RESTRICT="test"
 
 PATCHES=(
 	"${FILESDIR}"/${PN}-2.1.1-gcc12.patch
+	"${FILESDIR}"/${PN}-2.1.2-musl-stol.patch
```

Comment:
This line does not match with the patch file name `stol` != `strol`, which makes portage to fail:
```
 * Applying opencolorio-2.1.2-musl-stol.patch ...
/var/tmp/portage/media-libs/opencolorio-2.1.2/temp/environment: line 1642: /var/tmp/portage/media-libs/opencolorio-2.1.2/files/opencolorio-2.1.2-musl-stol.patch: No such file or directory
/var/tmp/portage/media-libs/opencolorio-2.1.2/temp/environment: line 1645: /var/tmp/portage/media-libs/opencolorio-2.1.2/files/opencolorio-2.1.2-musl-stol.patch: No such file or directory
 [ !! ]
 * ERROR: media-libs/opencolorio-2.1.2::gentoo failed (prepare phase):
 *   patch -p1  failed with /var/tmp/portage/media-libs/opencolorio-2.1.2/files/opencolorio-2.1.2-musl-stol.patch
```
 Anyway, I would still preffer to rename both to `strtol`.